tikv/kv.go

```diff
@@ -193,6 +194,13 @@ func WithPool(gp Pool) Option {
 	}
 }
 
+// WithPDAddrsAndTls set the pd addrs and tls for pdHttpClient.
+func WithPDAddrsAndTls(tlsConf *tls.Config, pdaddrs []string) Option {
```

Comment:
Suggest separating into two option functions.

```suggestion
func WithPDAddrs(pdaddrs []string) Option
func WithTLSConfig(tlsConf *tls.Config) Option
```

Reply:
I changed it to `WithPDHttpClient` instead of supporting 2 options, do you think this is appropriate?